templates/demo-store/server.ts

```diff
@@ -32,7 +32,7 @@ export default {
         throw new Error('SESSION_SECRET environment variable is not set');
       }
 
-      const waitUntil = (p: Promise<any>) => executionContext.waitUntil(p);
+      const waitUntil = executionContext.waitUntil.bind(executionContext);
```

Comment:
This change makes it easier to infer the type when using JSDocs because we are not wrapping `waitUntil` anymore